Inflow.Mobile/Inflow.Mobile/Inflow.Mobile/Views/LoginPage.xaml

```diff
@@ -7,78 +7,13 @@
              mc:Ignorable="d"
              x:Class="Inflow.Mobile.Views.LoginPage"
              Shell.NavBarIsVisible="False" FlowDirection="LeftToRight" >
-
+     
```

Comment:
Remove redundant empty line.

Reply:
I corrected everything according to the Xaml format